lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class MigrateDataToWorkField
+    QUERY = "data->>'employment_title' IS NOT NULL AND data->>'employment_title' <> ''".freeze
+
+    def run
+      Profile.where(QUERY).find_each do |profile|
```

Comment:
I verified via Blazer that this query is still workable with the default batch size of 10000.

Overall this migration will still run for a while though, have developed any best practices for these scenarios @maestromac (e.g. what to do when a DUS gets interrupted by a deploy or something)?

Reply:
One idea might be to adapt the QUERY to also exclude profiles where `work:` is populated, so if the job has to restart, the updated rows are skipped in the db rather than in line 7's "next if". This ensures an interrupted and restarted job can keep it's place and not needlessly instantiate all of the profile's it's not going to update.

Reply:
I would suggest what @jgaskins did recently by making these queue up individual sidekiq jobs in https://github.com/forem/forem/pull/13848/files

Reply:
Good idea @djuber, a quick test on Blazer makes this seem like it's feasible. 

@maestromac The individual updates are cheap here, so I don't worry too much about load (maybe wrongly). But the argument about disruptions is a good one.